frost-core/src/verifying_key.rs

```diff
@@ -76,7 +76,7 @@ where
         //                 h * ( z * B - c * A - R) == 0
         //
         // where h is the cofactor
-        let R = C::effective_nonce_element(signature.R);
+        let R = signature.R;
```

Comment:
Before this PR, a `Signature`'s `R` point could be either even or odd parity. When verifying, we were coercing `R` to be even, so that BIP340 verification logic would work. 

After this PR, the `Signature`'s `R` point is _always_ even parity (unless someone manually constructs one with an odd-parity outside the scope of this library). We no longer need to do this coercion during verification.